### PR TITLE
As a potential customer I can see a list of technologies that they are using, so that I know if their stack suits me

### PR DESCRIPTION
**Pull request by lspk129**
https://tree.taiga.io/project/troublemaker-website/task/38?kanban-status=932905

### CODE REVIEW

File: components/Technology/index.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+const Technology = () => (
+  <div>
+    <TechnologyHeader>
+      We use cutting-edge technologies
+    </TechnologyHeader>
+    <TechnologyContainer>
+      <TechnologyItem>
+        <img src="../../static/react_logo.png" alt="React" />
+        <h3>React Js</h3>
```
Comment (tmxyz) [inline components/Technology/index.js]: Change to just React.

File: components/Technology/index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      </TechnologyItem>
+      <TechnologyItem>
+        <img src="../../static/meteor_logo.png" alt="Meteor" />
+        <h3>Meteor Js</h3>
```
Comment (tmxyz) [inline components/Technology/index.js]: Change to just Meteor.

File: components/Body/index.js [hunk 3]
```diff
@@ -2,6 +2,7 @@ import styled from 'styled-components';
 
 const BodyContainer = styled.body`
   margin: 0;
+  font-family: roboto;
```
Comment (lspk129) [inline components/Body/index.js]: Fixed.

File: components/Hero/index.test.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import RaisedButton from 'material-ui/RaisedButton';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+
+
```
Comment (lspk129) [inline components/Hero/index.test.js]: Fixed.

File: components/Technology/TechnologyContainer/index.test.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (lspk129) [inline components/Technology/TechnologyContainer/index.test.js]: Fixed.

File: components/Technology/index.js (hunk 1 above)
Comment (lspk129) [inline components/Technology/index.js]: Fixed.

File: components/Technology/index.js (hunk 2 above)
Comment (lspk129) [inline components/Technology/index.js]: Fixed.

File: components/Technology/TechnologyItem/index.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  transition: all 0.5s ease-in-out;
+  display: grid;
+  align-items: center;
+  img {
+    width: 200px;
+    height: 200px;
+    align-self: center;
+    justify-self: center;
+  }
+  img[alt="Next"] {
```
Comment (lspk129) [inline components/Technology/TechnologyItem/index.js]: Fixed.

**Pull request merged**